tests/unit/CMakeLists.txt

```diff
@@ -246,6 +246,10 @@ add_grb_executables( outer outer.cpp
 	BACKENDS reference reference_omp hyperdags nonblocking
 )
 
+add_grb_executables( maskedOuter maskedOuter.cpp
+	BACKENDS reference
```

Comment:
I can do the implementation for the other backends than `reference_omp` if you want, that's pretty quick.

Reply:
Much obliged if you could complete the implementation. The dependencies necessary for the bsp1d seem to break something on my machine.

Reply:
Sure, I'll push it tomorrow

Reply:
(the tomorrow of today)